.github/workflows/ci.yml

```diff
@@ -324,6 +324,7 @@ jobs:
     with:
       project-name: Component[OpenTelemetry.Instrumentation.Http]
       code-cov-name: Instrumentation.Http
+      tfm-list: '[ "net462", "net8.0", "net9.0" ]'
```

Comment:
I need to add NET9 to the CI. I'm checking offline to confirm if this is the correct way to make that change.

Reply:
After #2313 gets merged, `net9.0` would run for all jobs by default. You won't need to configure for this individual job.